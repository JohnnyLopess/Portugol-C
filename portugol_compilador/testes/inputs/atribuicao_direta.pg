programa {
    inicio
        inteiro tab = 1
        escreva(tab)
}