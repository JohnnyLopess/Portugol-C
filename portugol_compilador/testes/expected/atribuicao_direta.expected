#include <stdio.h>
#include <string.h>

int main() {
    int tab = 1;
    printf("%d\n", tab);
    return 0;
}